tests/db/model/generatortemplate/utils.js

```diff
@@ -31,7 +31,7 @@ const GT_SIMPLE = {
     url: 'git+https://github.com/templates/template-generators.git',
   },
   connection: {
-    url: '{{baseTrustUrl}}/v1/instances/status/preview',
+    url: 'baseTrustUrl/v1/instances/status/preview',
```

Comment:
{{...}} must be allowed as part of connection.url and we must have tests for this